ext/OceananigansReactantExt/Grids/sharded_grids.jl

```diff
@@ -2,7 +2,7 @@ using Oceananigans.Architectures: architecture
 using Oceananigans.Grids: AbstractGrid
 using Oceananigans.OrthogonalSphericalShellGrids
 using Oceananigans.Grids: R_Earth, validate_lat_lon_grid_args, generate_coordinate, with_precomputed_metrics, validate_rectilinear_grid_args
-
+using Oceananigans.Grids: default_indices, validate_indices, offset_data
 import Oceananigans.Grids: zeros, StaticVerticalDiscretization
 import Oceananigans.Architectures: child_architecture
 
```

Comment:
This is a blind guess, let's see if it works:
```suggestion
import Oceananigans.Utils: instantiate

```